spec.html

```diff
@@ -6089,16 +6096,11 @@ <h1>
       </dl>
       <emu-alg>
         1. Assert: Type(_x_) is the same as Type(_y_).
-        1. If Type(_x_) is Undefined, return *true*.
-        1. If Type(_x_) is Null, return *true*.
-        1. If Type(_x_) is String, then
-          1. If _x_ and _y_ are exactly the same sequence of code units (same length and same code units at corresponding indices), return *true*; otherwise, return *false*.
-        1. If Type(_x_) is Boolean, then
-          1. If _x_ and _y_ are both *true* or both *false*, return *true*; otherwise, return *false*.
-        1. If Type(_x_) is Symbol, then
-          1. If _x_ and _y_ are both the same Symbol value, return *true*; otherwise, return *false*.
-        1. If _x_ and _y_ are the same Object value, return *true*. Otherwise, return *false*.
+        1. If _x_ is _y_, return *true*; otherwise, return *false*.
       </emu-alg>
+      <emu-note>
+        The specifics of what it means for _x_ and _y_ to be the same are detailed in <emu-xref href="#sec-identity"></emu-xref>.
```

Comment:
I would lightly prefer to leave the original text. We can add the note, but also having the cases doesn't hurt.

Reply:
I disagree. We can talk about it at editor call.

Reply:
+1 to @bakkot, I think readability is hurt by this brevity.